core-bundle/contao/languages/en/tl_page.xlf

```diff
@@ -320,6 +320,12 @@
       <trans-unit id="tl_page.hide.1">
         <source>Hide the page in the navigation menu.</source>
       </trans-unit>
+      <trans-unit id="tl_page.guests.0">
+        <source>Show to guests only</source>
```

Comment:
```suggestion
        <source>Hide in navigation if logged in</source>
```

The “Show to guests only” label is misleading IMO.

Reply:
I still disagree with this change (https://github.com/contao/contao/issues/7984#issuecomment-2624460239)

Reply:
I see it that way: One possibility is that a user already understood this feature correctly, in this case they should not be confused by “Hide in navigation if logged in” as this is exactly what this setting does.

The other possibility is that they misunderstood the feature in which case the changed label only can improve the understanding of the user.

Reply:
Alright then 😁